pinot-common/src/main/java/org/apache/pinot/common/utils/ZkStarter.java

```diff
@@ -51,17 +52,18 @@ private ZookeeperInstance(PublicZooKeeperServerMain serverMain, String dataDirPa
   static class PublicZooKeeperServerMain extends ZooKeeperServerMain {
     @Override
     public void initializeAndRun(String[] args)
-        throws QuorumPeerConfig.ConfigException, IOException {
+        throws QuorumPeerConfig.ConfigException, IOException, AdminServer.AdminServerException {
```

Comment:
Where is `AdminServer.AdminServerException` thrown from?

Reply:
From ZooKeeperServerMain, it's added in zookeeper 3.5.x